RadarSDK/RadarDelegateHolder.m

```diff
@@ -34,8 +34,8 @@ - (void)didReceiveEvents:(NSArray<RadarEvent *> *)events user:(RadarUser *)user
 
     for (RadarEvent *event in events) {
         [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelInfo
-                                           message:[NSString stringWithFormat:@"📍 Radar event received | type = %@; link = https://radar.com/dashboard/events/%@",
-                                                                              [RadarEvent stringForType:event.type], event._id]];
+                                           message:[NSString stringWithFormat:@"📍 Radar event received | type = %@; replayed= %d link = https://radar.com/dashboard/events/%@",
```

Comment:
We should bring this change to Android for consistency

Reply:
Great point, done. https://github.com/radarlabs/radar-sdk-android/pull/312/commits/8535265721ea18c2f7953be368d5afac748c01c1